src/collective/glossary/browser/views.py

```diff
@@ -62,6 +65,31 @@ def terms(self, letter):
 
         return self.get_entries()[letter]
 
+    @cached_property
+    def use_az_toolbar(self):
+        """Use the A-Z toolbar.
+
+        When there are many entries, we show an A-Z toolbar.
+        Initially no entries are shown.  Once you click on one of the letters,
+        the entries for this letter are shown.
+
+        For small glossaries this makes no sense, so we show them all.
+        You can override the default maximum with an environment variable.
+        To never show the toolbar, set this to a negative number (-1).
+        """
+        maximum_without_az_toolbar = api.portal.get_registry_record(
+            IGlossarySettings.__identifier__ + ".maximum_without_az_toolbar"
+        )
```

Comment:
Please provide a default value. Without upgrading with your upgrade step, this throws an plone.api.exc.InvalidParameterError.
```suggestion
        maximum_without_az_toolbar = api.portal.get_registry_record(
            IGlossarySettings.__identifier__ + ".maximum_without_az_toolbar",
            default=DEFAULT_MAXIMUM_WITHOUT_AZ_TOOLBAR
        )
```

Reply:
Good point.  Fixed, thanks.